### PR TITLE
courses: smoother markdown dark mode (fixes #4911)

### LINKED ISSUE #4911: make webview darkmode compatible
[Screen_recording_20241223_162222.webm](https://github.com/user-attachments/assets/85359217-9c5f-4dd3-9cc5-fe94eac32cad)

### PR DESCRIPTION
**Pull request by Avinash-Codes**
fixes #4911

### CODE REVIEW

File: app/src/main/java/org/ole/planet/myplanet/ui/viewer/WebViewActivity.kt [hunk 1]
```diff
@@ -52,17 +71,58 @@ class WebViewActivity : AppCompatActivity() {
...
+            """.trimIndent(),
+                        null
+                    )
+                }
             }
+
         }
     }
 
+
```
Comment (Okuro3499) [inline app/src/main/java/org/ole/planet/myplanet/ui/viewer/WebViewActivity.kt:119]: remove the double space here

File: app/src/main/java/org/ole/planet/myplanet/ui/viewer/WebViewActivity.kt [hunk 2]
```diff
@@ -52,17 +71,58 @@ class WebViewActivity : AppCompatActivity() {
 
             override fun onPageFinished(view: WebView, url: String) {
                 super.onPageFinished(view, url)
+
+                // Detect if dark mode
+                val nightModeFlags = resources.configuration.uiMode and android.content.res.Configuration.UI_MODE_NIGHT_MASK
+                if (nightModeFlags == android.content.res.Configuration.UI_MODE_NIGHT_YES) {
+                    // Enable dark mode
```
Comment (Okuro3499) [inline app/src/main/java/org/ole/planet/myplanet/ui/viewer/WebViewActivity.kt:78]: remove the comments

**Review by Okuro3499 — CHANGES_REQUESTED** [review]
@Avinash-Codes 
could also make the top bar also darkmode compatible
Screenshot_20241223_185132

**Comment by Avinash-Codes** [comment]
> @Avinash-Codes could also make the top bar also darkmode compatible Screenshot_20241223_185132

done

**Review by Okuro3499 — APPROVED** [review]

**Review by dogi — APPROVED** [review]

**Pull request merged**